test/libyul/yulOptimizerTests/loadResolver/keccak_fail2.yul

```diff
@@ -0,0 +1,15 @@
+{
+    mstore(0, 10)
+    // Although, this can be evaluated, we're skipping it.
```

Comment:
That's not what the expectation looks like right now, though, is it?

Reply:
Oops. Fixed it. It was a copy and paste issue.